### PR TITLE
[CognitiveServices] Support getting account types (kinds).

### PR DESCRIPTION
**Pull request by yangyuan**
Add `Get-AzureRmCognitiveServicesAccountType`  cmdlet to allow customers to get available kind names.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

Design Review:
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/177

### CODE REVIEW

File: src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/help/AzureRM.CognitiveServices.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ Gets the API keys for an account.
 ### [Get-AzureRmCognitiveServicesAccountSkus](Get-AzureRmCognitiveServicesAccountSkus.md)
 Gets the available SKUs for an account.
 
+### [Get-AzureRmCognitiveServicesAccountType](Get-AzureRmCognitiveServicesAccountType.md)
+{{Fill in the Synopsis}}
```
Comment (MiYanni) [inline src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/help/AzureRM.CognitiveServices.md]: Description needs filled in.

File: src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/help/Get-AzureRmCognitiveServicesAccountType.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+```
+
+### GetAccountTypes
+```
+Get-AzureRmCognitiveServicesAccountType [-Location <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
```
Comment (MiYanni) [inline src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/help/Get-AzureRmCognitiveServicesAccountType.md]: Description needs filled in.

File: src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/help/Get-AzureRmCognitiveServicesAccountType.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+external help file: Microsoft.Azure.Commands.Management.CognitiveServices.dll-Help.xml
+Module Name: AzureRM.CognitiveServices
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmCognitiveServicesAccountType
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```
Comment (MiYanni) [inline src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/help/Get-AzureRmCognitiveServicesAccountType.md]: Synopsis needs filled in.

File: src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/help/Get-AzureRmCognitiveServicesAccountType.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
+---
+external help file: Microsoft.Azure.Commands.Management.CognitiveServices.dll-Help.xml
+Module Name: AzureRM.CognitiveServices
+online version:
```
Comment (MiYanni) [inline src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/help/Get-AzureRmCognitiveServicesAccountType.md]: Online version needs filled in (should be part of the StaticAnalysis failures).

**Comment by MiYanni** [comment]
@yangyuan Yes. CmdletsToExport in `Az.CognitiveServices.psd1` should have the same list as what is in `AzureRM.CognitiveServices.psd1`.

**Comment by MiYanni** [comment]
@yangyuan Ping

**Comment by yangyuan** [comment]
@MiYanni, sorry for late response, got stuck in some other tasks.

Will try to finish this RP today,

File: src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/CognitiveServicesAccountBaseCmdlet.cs [hunk 5]
```diff
@@ -32,6 +32,7 @@ public abstract class CognitiveServicesAccountBaseCmdlet : AzureRMCmdlet
         protected const string CognitiveServicesAccountNounStr = "AzureRmCognitiveServicesAccount";
```
Comment (MiYanni) [inline src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/CognitiveServicesAccountBaseCmdlet.cs]: This has to be changed to:
```suggestion
         protected const string CognitiveServicesAccountNounStr = ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CognitiveServicesAccount";
```
If not, this won't work properly for `Az`.

File: src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/Commands.Management.CognitiveServices.csproj [hunk 6]
```diff
@@ -72,6 +73,7 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <None Include="Az.CognitiveServices.psd1" />
```
Comment (MiYanni) [inline src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/Commands.Management.CognitiveServices.csproj]: Remove this.

File: src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/CognitiveServicesAccountBaseCmdlet.cs (hunk 5 above)
Comment (yangyuan) [reply src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/CognitiveServicesAccountBaseCmdlet.cs]: coll, also this `suggest change` feature is cool.

**Pull request merged**